graphs/breathfirstsearch/breathfirstsearch.go

```diff
@@ -1,73 +1,31 @@
 package breathfirstsearch
 
-// GetIdx ...
-func GetIdx(target int, nodes []int) int {
-	for i := 0; i < len(nodes); i++ {
-		if nodes[i] == target {
-			return i
-		}
-	}
-	return -1
-}
+/*
 
-// NotExist ...
-func NotExist(target int, slice []int) bool {
-	for i := 0; i < len(slice); i++ {
-		if slice[i] == target {
-			return false
-		}
-	}
-	return true
-}
+	Breadth First Search
+	https://en.wikipedia.org/wiki/Breadth-first_search
+
+*/
 
```

Comment:
The comments are written immediately before the function definition for better parsing of automatic documentation
```suggestion
```

Reply:
This should've been caught on our linting - @cclauss any idea how we can add this in the linting configuration?